### PR TITLE
fix group spreadsheet

### PR DESCRIPTION
**Pull request by anistouri**

### CODE REVIEW

**Comment by Meklo** [comment]
I changed `editableCondition` to `crossValidation` and a nested object `requiredOn` to make it easier to grasp

File: src/components/spreadsheet/utils/cell-renderers.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 export const BooleanCellRenderer = (props) => {
-    const isChecked = props.value;
+    const isChecked = Boolean(props.value);
```
Comment (Meklo) [inline src/components/spreadsheet/utils/cell-renderers.js:67]: Not linked to the initial issue but fixes a warning when updating a boolean value

File: src/components/spreadsheet/utils/equipment-table-editors.js [hunk 2]
```diff
@@ -133,7 +162,7 @@ export const NumericalField = forwardRef(
         function renderNumericText() {
             return (
                 <TextField
-                    value={value}
+                    value={value || ''}
```
Comment (Meklo) [inline src/components/spreadsheet/utils/equipment-table-editors.js:165]: Not linked to the initial issue but fixes a warning when filling an initially empty field

**Review by Walid-Sahnoun — APPROVED** [review]
review ok
test ok

**Review by archendel — APPROVED** [review]
test: OK
code: OK

**Pull request merged**